exec_test.go

```diff
@@ -494,3 +493,105 @@ func TestCommandCombinedOutput(t *testing.T) {
 		})
 	}
 }
+
+func TestCommandStdin(t *testing.T) {
+	d := newTestDistro(t, jammyRootFs)
+
+	const (
+		readFromPipe int = iota + 1
```

Comment:
any reason to just default to this one, having it set to 0, (so just `iota` and not +1), which allows then to not specify it explicitely in the 3 first tests, as this is the expected default, and we want to focus on the specific of the test itself (standard, funny characters, closing early). More on that on my next comment.

Reply:
Well neither seemed like a better default than the other, so I made neither of them default 😅. In the end three out of four use pipes so I suppose that became the de facto standard.

Anyways, I removed the +1.

Reply:
yeah, you are right, I was first reading the function on testing StdinPipe primarly (mostly because it was the biggest change of this PR), but the name is more opened as testing StdIn, so I am a little bit more opened. As you said, the fact that you focus primarly on this for the test make it the standard and I think this is fine.

You can maybe then change "standard" to "from pipe"? Then, the rest (the fact that we are using pipes rather than buffer) is an test detail… WDYT?

Reply:
I agree, `from pipe` is more descriptive, as using pipes is no more standard than using buffers.